extensions-core/kafka-extraction-namespace/src/main/java/io/druid/server/namespace/KafkaExtractionManager.java

```diff
@@ -83,8 +79,7 @@
   // Bindings in KafkaExtractionNamespaceModule
   @Inject
   public KafkaExtractionManager(
-      @Named("namespaceVersionMap") final ConcurrentMap<String, String> namespaceVersionMap,
-      @Named("renameKafkaProperties") final Properties kafkaProperties
+      final Properties kafkaProperties
```

Comment:
This will inject `Properties` from the properties module, not the kafka properties.